packages/discord.js-utilities/src/lib/PaginatedMessage.ts

```diff
@@ -355,7 +370,10 @@ export class PaginatedMessage {
 		{
 			id: '⏹️',
 			run: async ({ response, collector }) => {
-				await response.reactions.removeAll();
+				if ((response.channel as TextChannel).permissionsFor(response.client.user?.id || '')?.has('MANAGE_MESSAGES')) {
+					await response.reactions.removeAll();
+				}
+
```

Comment:
Similar change as above. Don't typecast, that's extremely unsafe and this will crash in DMs because `permissionsFor` is not a method on `DMChannel`